frontend/src/pages/account-settings/styles.module.scss

```diff
@@ -44,3 +40,11 @@
   margin: 38px 0 71px;
   background-color: var(--white);
 }
+
+.profile-link-active {
+  border-bottom: 2px solid var(--yellow);
+}
+
+.active-link {
+  border-bottom: 2px solid var(--yellow);
+}
```

Comment:
these two classes duplicate each other styles - and actually it looks like it could be an active link styling for common `Link` component - let's declare and use it in `Link` component and for links that should have separate _active_ styling lets leave `activeClassName`